cmd/plume/specs.go

```diff
@@ -152,6 +156,56 @@ var (
 		},
 	}
 	specs = map[string]channelSpec{
+		"dev": channelSpec{
+			BaseURL: "gs://users.developer.core-os.net/" + os.Getenv("USER") + "/boards",
+			Boards:  []string{"amd64-usr", "arm64-usr"},
+			Destinations: []storageSpec{storageSpec{
+				BaseURL:       "gs://users.developer.core-os.net/" + os.Getenv("USER") + "/releases",
+				NamedPath:     "current",
+				VersionPath:   true,
+				DirectoryHTML: true,
+				IndexHTML:     true,
+			}},
+			GCE: gceSpec{
+				Project:     "coreos-gce-testing",
+				Family:      "coreos-dev",
+				Description: "CoreOS Container Linux development image",
+				Image:       "coreos_production_gce.tar.gz",
+				Publish:     "coreos_production_gce.txt",
+				Limit:       2,
+			},
+			AWS: awsSpec{
+				BaseName:        "ContainerLinuxDev",
+				BaseDescription: "CoreOS Container Linux development image",
+				Prefix:          "coreos_production_ami_",
+				Image:           "coreos_production_ami_vmdk_image.vmdk.bz2",
+				Partitions: []awsPartitionSpec{
+					awsPartitionSpec{
+						Name:         "AWS West",
+						Profile:      "default",
```

Comment:
I think requiring the name to be `coroes-dev` is slightly safer, but I don't feel strongly.